slack_sdk/web/legacy_client.py

```diff
@@ -2994,6 +2997,160 @@ def files_upload(
             kwargs["content"] = content
             return self.api_call("files.upload", data=kwargs)
 
+    def files_upload_v2(
```

Comment:
I'm am not sure what our approach is to the `legacy_client`, as a dev I would expect no new features to be added to something named `legacy`

Could you let me know what approach regarding new features we maintain towards this code?

Reply:
@WilliamBergamin Indeed, we no longer need to support the legacy one for new features, and it's fine to stop generating code for it in the future if we want to do so. however, regarding this (files.upload v2 migration), I would like to make it available for legacy client users.

Reply:
Sounds good 👍 so this part of the code is what the docs refer to as `maintenance mode`